### ISSUE TITLE
Data for African countries

### ISSUE THREAD

**Issue by madsnedergaard** [electricitymap/electricitymap-contrib#3712]
[Ember](https://ember-climate.org/) has just released a report that gathers available electricity data for all African countries 🌍 

It would be really cool to go through the report and see if any of the data sources have real-time data available!

Report: https://ember-climate.org/project/africa-electricity-data/
<img width="583" alt="Screenshot 2022-01-26 at 18 50 25" src="https://user-images.githubusercontent.com/3296643/151218619-eb81889c-0fa9-4665-88fb-56b4ac3068e7.png">

**Comment by alixunderplatz** [comment]
I'd like to link an older collection of links for African Power Pools / TSO-cooperations in #1218, just in case somebody wants to take a look :)
If I remember correctly, South African Power Pool SAPP had some price data published. Probably that is also linked in the document above :)

**Comment by IV1T3** [comment]
The presentation contains some interesting metrics regarding the data availability for each country.
The most important metrics are "Publishing lag", "Time granularity" and "Fuel breakdown" each ranked from 5 (best) to 0 (worst).

The table below contains the most promising countries based solely on these metrics sorted in descending order:

| Country | Publishing lag | Time granularity | Fuel breakdown |
|---------|----------------|------------------|----------------|
|     Nigeria    |        5 (<= 1 week)        |         4 (weekly or better)         |      1 (no fuel split)          |
|     South Africa    |        4 (<= 1 month)        |         5 (hourly)         |      3 (Some)          |
|    ---    |        ---        |         ---         |     ---         |
|    Kenya     |       3 (<= 2-6 month lag)         |       3 (monthly)           |        4 (Ember-level fuel split)        |
|    Namibia     |       3 (<= 2-6 month lag)         |       3 (monthly)           |        2 (high level split)        |
|    Burundi     |       3 (<= 2-6 month lag)         |       3 (monthly)           |        2 (high level split)        |

However, looking at the last three countries, I don't know whether we can effectively use data with such a granularity or lag.

**Comment by IV1T3** [comment]
Interesting supply side source from a south african utilitiy:
https://www.eskom.co.za/dataportal/supply-side/station-build-up-for-the-last-7-days/

**Comment by VIKTORVAV99** [comment]
@Kongkille now that historical view is launched (any minute now) it would make sense to build parsers for these even if the data lag is several days to months right?

We wouldn't get any realtime data but at least we would have the historical data, it's better than nothing right?

**Comment by Kongkille** [comment]
> @Kongkille now that historical view is launched (any minute now) it would make sense to build parsers for these even if the data lag is several days to months right?
> 
> We wouldn't get any realtime data but at least we would have the historical data, it's better than nothing right?

Absolutely, however at this time we do not have the functionality of parsing data other than hourly granularity. All aggregated data shown in the historical view is based on hourly data.

In the future, it would be amazing to have the ability to parse daily, monthly, yearly data directly
